lib/src/content/wolt_modal_sheet_animated_switcher.dart

```diff
@@ -411,12 +397,9 @@ class _WoltModalSheetAnimatedSwitcherState
     );
   }
 
-  PaginatingWidgetsGroup _createOutgoingWidgets(
-    AnimationController animationController,
-    PaginatingWidgetsGroup currentWidgetsToBeOutgoing,
-  ) {
-    const animatedBuilderKey =
-        ValueKey(WoltModalSheetPageTransitionState.outgoing);
+  PaginatingWidgetsGroup _createOutgoingWidgets(AnimationController animationController,
+      PaginatingWidgetsGroup currentWidgetsToBeOutgoing,) {
```

Comment:
```suggestion
      PaginatingWidgetsGroup currentWidgetsToBeOutgoing) {
```

Extra comma. After fixing please execute `dart format .` to update the format.

Reply:
Out of curiosity, why would comma not be needed here? I think it needs to be there since it makes it easier to read. After dart format, it correctly shows.

<img width="505" alt="image" src="https://github.com/woltapp/wolt_modal_sheet/assets/13782003/c9de6819-6570-4cda-bb94-2c36b0c66cf0">

Reply:
Ah.. looks good. Without format it was off.